tests/integration/sync/it_sync_location_test.py

```diff
@@ -65,24 +66,24 @@ def test_sync_location(db_session):
     )
     db_session.add(location)
     db_session.commit()
-    synced_location = SyncedLocation(location).sync(
-        Poster(
+    SyncedLocation(
+        location=location,
+        poster_sync=Poster(
             url="http://localhost:{port}".format(port=port)
-        )
-    )
-    row = db_session.query(Location).get(synced_location.id)
-    assert(
-        row.id == 100 and
-        row.name == "Coco Bongo" and
-        row.code == "C" and
-        row.company_id == 50 and
-        row.country == "United States" and
-        row.region == "East Coast" and
-        row.city == "Edge City" and
-        row.address == "Blvd. Kukulcan Km 9.5 #30, Plaza Forum" and
-        row.longitude == 21.1326063 and
-        row.latitude == -86.7473191 and
-        row.type == "L" and
-        row.status == "open" and
-        row.comment == "Nightclub from a famous movie"
-    )
+        ),
+        db_session=db_session
+    ).sync()
+    row = db_session.query(Location).get(location.id)
+    assert row.id == 100 
+    assert row.name[0] == "Coco Bongo"
```

Comment:
@paulodamaso What does mean `[0]`, index for what?

Reply:
@vladarefiev Strangely some `row` attrributes are being retrieved as tuples from the database so I have to get the first item of them to the test work

Reply:
@paulodamaso It's not strange. It says that here a mistake.

Reply:
@paulodamaso That sould alarm to you, that `id` and `comment` is not a tuple, but rest are tuples. That's stange, smth is wrong

Reply:
@vladarefiev Please I need you to be more assertive in your comments, I am having a terrible difficulty in understanding what you want to say to me. Is that strange or not? I really couldn't find what is wrong here and why this is happening, so that's the only way I made it run properly. Do you have any lead about the problem (if any) could be?

Reply:
@paulodamaso I was surpriced to see:
```
assert row.id == 100 
assert row.name[0] == "Coco Bongo"
```
I don't know that is actually wrong, but smth is wrong.
`row = db_session.query(Location).get(location.id)`
`row` is Location instance which is fetched from db. Location has attributes, such as `id`, `name`. For some reason a few attr contains tuples. That's a consequence of an error. Smth is wrong when you put data to DB. 
Breathly I could not find a mistake.

Reply:
@paulodamaso Test should look like 
```

    assert row.id == 100 
    assert row.name == "Coco Bongo"
    assert row.code == "C"
    assert row.company_id == 50
    assert row.country == "United States"
    assert row.region == "East Coast"
    assert row.city == "Edge City"
    assert row.address == "Blvd. Kukulcan Km 9.5 #30, Plaza Forum"
    assert row.longitude == 21.1326063
    assert row.latitude == -86.7473191
    assert row.type == "L"
    assert row.status == "open"
    assert row.comment == "Nightclub from a famous movie"
```

Reply:
@vladarefiev Yes, I 'm puzzled by this too. Take a look at this debug steps:
- `SyncedLocation.sync` uses two instances of `Location`, one from database and one from `Poster`. First of all I thought that `Poster` instance was giving me the wrong value, but `loc["name"]` is returned as `Coco Bongo`, so it is coming right from `Poster` service (I even checked the type returned with `type(loc["name"])`, and it returns `str`)
- The value stored is being stored in database as a touple. I've print it on screen and `print(row.name)` returns `('Coco Bongo',)`, while `print(type(row.name))` returns `<class 'tuple'>`. I'll take a deeper look at it.

Reply:
@paulodamaso 
after `print(row.name) returns ('Coco Bongo',)` it became clear. 

You save to db a tupe instead of what you want.
`location.status = loc["status"],` 
comma is a syntax for tuples.
everywhere should be the same
`location.comment = loc["comment"]`

here is why `comment` is `str`, but `status` is `tuple`

Reply:
@paulodamaso also here is bad naming, difficult to understand what is location, loc. pls fix also naming